packages/app/src/cli/services/build/extension.ts

```diff
@@ -52,6 +52,25 @@ export async function buildThemeExtensions(options: ThemeExtensionBuildOptions):
     stdout: options.stdout,
     stderr: options.stderr,
   })
+
+  if (environment.local.useThemeBundling()) {
+    await Promise.all(
+      options.extensions.map(async (extension) => {
+        options.stdout.write(`Bundling theme extension ${extension.localIdentifier}...`)
+        const files = await path.glob(path.join(extension.directory, '/**/*'))
+
+        await Promise.all(
+          files.map(function (filepath) {
+            if (!(filepath.includes('.gitkeep') || filepath.includes('.toml'))) {
+              const relativePath = path.relative(extension.directory, filepath)
+              const outputFile = path.join(extension.outputBundlePath, relativePath)
+              return file.copy(filepath, outputFile)
+            }
```

Comment:
@Arkham I guess you meant this change? We shouldn't be bundling on every `build`, this method should just execute the `execThemeCheckCLI`.

We should have a different `bundleThemeExtension` method for this that's called only for deploy at `packages/app/src/cli/services/deploy/bundle.ts`